test/integration/basic_test.go

```diff
@@ -181,11 +178,25 @@ func FeatureContext(s *godog.Suite) {
 }
 
 func TestMain(m *testing.M) {
+	flag.StringVar(&givenArgs, "minishift-args", "", "Arguments to pass to minishift")
```

Comment:
Cool, now we can handle all potential flags. Personally I'd put the command line parsing into a separate function. But this will work.